### ISSUE TITLE
Update Page Output to Use <main> tag for wp-page 

### ISSUE THREAD

**Issue by thezenmonkey** [elementor/elementor#17578]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
In an effort to increase accessibility better landmarks are needed in the default HTML output of Elementor. <header> and <footer> tags can be added page headers and footers but we're still missing the <main> tag

**Describe the solution you'd like**
The data-elementor-type="wp-page" element should be output with a <main> tag instead of a generic <div> to provide better landmarks for accessibility

**Describe alternatives you've considered**

**Additional context**

**Comment by rami-elementor** [comment]
@thezenmonkey 

This feature already exists in Elementor for a very long time.

You can select the wrapping HTML tag for sections / columns :

![main-section](https://user-images.githubusercontent.com/92088692/153729750-c213b2bd-3ad3-4206-9207-90e995ae5123.png)

Or, if you are using the Builder, navigate to the page settings panel (the cog icon in the bottom left corner), and choose your HTML tag:

![page-level](https://user-images.githubusercontent.com/92088692/153729795-f16c1685-048a-45ac-8694-207a5e13384d.png)

**Comment by thezenmonkey** [comment]
That feature is at the section level. For accessibility requirements you can only have one <main> element per page. Therefore it should be the container for the Elementor sections

**Comment by rami-elementor** [comment]
@thezenmonkey 

Again, you can set this at the container level.

In the second screenshot I used the **Theme Builder** to create a template for all Single Posts. The entire area can receive custom HTML tags.

**Comment by SolunaCG** [comment]
I´m not sure if I understand this correctly. Maybe @rami-elementor can elaborate once more. 
So from what I´ve experienced with my own websites I can say that I always have the problem that I can assign HTML tags to header and footer (which is great) but not to the actual page which is what this thread is about. And I agree that every wordpress page that was built with Elementor should always have the "main" HTML element assigned by default. Just like it already is with the header and footer. 
I know that we can assign these HTML tags to containers or sections but that doesn´t solve the main problem here that the "main" tag should be applied to the top-level element of the content of the page. Again, just like it is already with the header and footer.

![no-main](https://github.com/elementor/elementor/assets/114682524/166ae704-7837-42bd-9b4b-a57275bffca4)

As you can see when editing either header or footer we can go in the settings by clicking on the gear icon on the bottom left and then assign the HTML tag. 

![footer-settings](https://github.com/elementor/elementor/assets/114682524/4296819d-6665-4fdb-9705-c831bda62d4d)

However, for the actual page that doesn´t work and I fully agree that this should be possible, too! 

Imo it should be possible to assign it here in the actual "Page Settings" (again, just like it already is with header and footer):

![page-settings-without-html-tag](https://github.com/elementor/elementor/assets/114682524/cbffebe5-8772-4a85-8181-e7ab1f53337c)

**Comment by rami-elementor** [comment]
@SolunaCG

The page has no wrapper HTML tag. Use a single container (with a wrapping `<main>` tag), and nest all the elements inside that container.

**Comment by SolunaCG** [comment]
Not sure if I did something wrong or if it´s just the way it is but when I just put my whole page content in another container and give that wrapper-container the "main" tag then it´s still not as it should be. Then it´s like this:

```
<header>
<div data-elementor-type="wp-post">
    <main>
<footer>
```

**Comment by annelyse** [comment]
I totally agree; the selection of the tag for page is missing !!

if you add a container and nested the content inside, the structure is not good
<header>
<div> 
  <main>
     content
  </main> 
</div>
<footer>

and if you add a template and apply to the page, you have an extra div ..;
<header>
<main> 
  <div>
     content
  </div> 
</main>
<footer>
